app/models/service_template_provision_task.rb

```diff
@@ -117,48 +117,44 @@ def do_post_provision
   end
 
   def deliver_to_automate(req_type = request_type, _zone = nil)
-    if self.class::AUTOMATE_DRIVES
```

Comment:
Can we also remove the constant above?  Or do subclasses still use this?

Reply:
The parent class still uses this https://github.com/ManageIQ/manageiq/blob/master/app/models/miq_request_task.rb#L142